flamingo/scripts/bh_jet_energies.py

```diff
@@ -0,0 +1,152 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats as stats
+
+from swiftsimio import load
+
+from unyt import mh, cm, Gyr
+from matplotlib.colors import LogNorm
+from matplotlib.animation import FuncAnimation
+
+# Set the limits of the figure.
+mass_bounds = [1e5, 3e10]
+value_bounds = [1e58, 1e63]
+bins = 25
+def_value = -1.0
+
+
+def get_data(filename):
+
+    data = load(filename)
+
+    masses = data.black_holes.subgrid_masses.to("Msun")
+
+    try:
+        values = data.black_holes.injected_jet_energies.astype(np.float64).to("erg")
+    except:
+        values = np.zeros(np.size(masses))
```

Comment:
Does this actually work on a snapshot that is missing the jet energies? The `except` case is a plain `numpy` array, while the normal case is a `unyt_array`. I expect that this will give problems down the line.

Suggestion:
```suggestion
        values = unyt.unyt_array(np.zeros(masses.shape), dtype=np.float64, units="erg")
```

Of course, you will then need to properly `import unyt`.

Reply:
I have implemented this and tested it out on a snapshot from the master branch (albeit a colibre one, since I don't have any flamingo ones). It correctly creates the plots, but they are empty, as they should be.